src/aead/shift.rs

```diff
@@ -28,14 +29,14 @@ where
             transform(input)
         };
         let output = <&mut [u8; BLOCK_LEN]>::try_from(&mut in_out[i..][..BLOCK_LEN]).unwrap();
-        *output = *block.as_ref();
+        *output = block;
```

Comment:
Codecov is right to complain about the lack of test coverage for this function in the `coverage` job in CI. That's something to be addressed separately.